src/style.css

```diff
@@ -1 +1,110 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap");
+
+html,
+body,
+div,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+a {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
```

Comment:
In this part above you've reset default padding, margin for a lot of tags, but why if you don't use them on your page? Also why are you reset the border? All of this tags not have default border